### DISCUSSION TITLE
Slack Invite

### DISCUSSION THREAD

**Discussion by Enragedsaturday** [code4recovery/12-step-meeting-list#581] [General]
Hi folks,
I've emailed info/devs email addresses a couple times to no avail. I'm trying to join the team.

Thanks,
Ben

Email: benjaminspringer@protonmail.com

**Comment by tech2serve** [comment]
My personal apologies, Ben. I’ll need to ry and figure out what is going on with our email. I’m not seeing any messages from you, but will email you.
